actions/FirefallClient.js

```diff
@@ -57,7 +69,7 @@ class FirefallClient {
       return response;
     } catch (error) {
       logger.error('Failed generate request:', error);
```

Comment:
Let's change the logger statement to 'Failed generating results' to sound more natural.